pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -90,13 +96,40 @@ If you have a screen attached to your Raspberry Pi and notice that there is a sm
 
 ### Unable to flash a board that was previously running L4T 28.x
 
-With {{ $names.os.lower }} 2.47, the [NVIDIA Tegra Linux Driver Package][l4t] (L4T) was upgraded to version 32.2 from a previous version of 28.x. Trying to flash a board that was previously running L4T 28.x with an SD card containing L4T 32.2 will fail. This failure happens because the new 32.2 kernel is incompatible with the NVIDIA eMMC partitions from L4T 28.x.
+With {{ $names.os.lower }} 2.47, the [NVIDIA Tegra Linux Driver Package][l4t] (L4T) was upgraded to version 32.2 from a previous version of 28.x. Trying to flash a board that was previously running L4T 28.x with an SD card containing L4T 32.2 or newer will fail. This failure happens because L4T 32.2 and newer kernels are incompatible with the NVIDIA eMMC partitions from L4T 28.x, and vice-versa.
 
 To resolve, either:
 
 * Perform a [self-service update][self-service-update] to update {{ $names.os.lower }} to >=2.47.
 * Update the Tegra partitions on the eMMC by [downloading the L4T Driver Package][l4t-download] (BSP), unpack it, put the board in recovery mode and execute `sudo ./flash.sh jetson-tx2 mmcblk0p1`. Once this completes, you can reboot the board with the SD card inserted and flash the board as normal. You only need to perform this process once.
 
+### Are L4T 28.4 and 28.5 releases supported in BalenaOS?
+
+The L4T versions 28.4 and 28.5 did not follow the usual incremental release process, instead these version were released in July 2020 and July 2021 respectively, well after the first iteration of L4T 32.1 was released.
+Since BalenaOS uses the OE4T/meta-tegra yocto repository and only updates to incremetal versions of L4T, BalenaOS images based on 28.4 and and 28.5 which were released after L4T 32.1 are not available in the Balena cloud.
+
+### Can a hostOS update from an older L4T 28.X based release to a newer L4T 32.Y be interrupted? Can I boot back into the old OS?
+
+Starting with L4T 32.1 the NVidia partition layout has changed, so did the firmwares and bootloaders stored on the raw partitions. The hostOS update process thus needs to re-create all the NVidia partitions that were modified in the newer L4T, to allow for the newer bootloaders and kernels to run. L4T 28.X partition layout is not compatible with the L4T 32.X format and vice-versa, which means that a newer kernel cannot boot with older partition format, nor can an older kernel boot with the new layout and contents. Therefore, the hostOS update process should not interrupted on the Jetson family of boards, otherwise the device may not boot. Please ensure the device is powered correctly or has sufficient battery, the internet connectivity is stable and that it is not powered-off or rebooted manually during the hostOS update.
```

Comment:
```suggestion
Starting with L4T 32.1 the NVidia partition layout has changed as well as the firmware and bootloaders stored on the raw partitions. The hostOS update process thus needs to re-create all the NVidia partitions that were modified in the newer L4T, to allow for the newer bootloaders and kernels to run. The L4T 28.x partition layout is not compatible with the L4T 32.x format and vice-versa, which means that a newer kernel cannot boot with the older partition format, nor can an older kernel boot with the new layout and contents. Therefore, the hostOS update process should not interrupted on the Jetson family of boards, otherwise the device may not boot. Please ensure the device is powered correctly or has sufficient battery, the internet connectivity is stable and that it is not powered-off or rebooted manually during the hostOS update.
```